### PR TITLE
update to `sf` lesson

### PR DESCRIPTION
**Pull request by jeanetteclark**
swapped out data, made a more complete example, and added a section on leaflet

### CODE REVIEW

**Comment by jeanetteclark** [comment]
also, you can bypass rawgit hosting if you link directly to the html like this:
https://jeanetteclark.github.io/spatial-analysis-R/intro_to_sf.html

(and also that might be the best place to review my edits)

File: intro_to_sf.Rmd [hunk 1]
```diff
@@ -78,61 +82,125 @@ You'll notice right away that these two objects are being plotted differently. T
...
+
+```{r}
+st_crs(ak_shp_sf)
+```
+
+This is pretty confusing looking. Withinout getting into the details, that long string says that this data is projected in an Alaska Albers projection with a NAD83 datum. A more convenient way to reference `crs` quickly is by using the EPSG code, a number that represents a standard projection and datum. You can check out a list of (lots!) of EPSG codes [here](http://spatialreference.org/ref/epsg/?page=1). 
+
+Even though it is not listed above, the Alaska Albers projection has an EPSG code 3338.
+
+You will often need to transform your geospatial data from one coordiante system to another. The `st_transform` function does this quickly for us.
```
Comment (jamiecmontgomery) [inline intro_to_sf.Rmd]: coordinate is misspelled.
```suggestion
You will often need to transform your geospatial data from one coordinate system to another. The `st_transform` function does this quickly for us.
```

**Review by jamiecmontgomery — CHANGES_REQUESTED** [review]
Looks good! I'm just asking for this spelling change to test out the review process. We might want to add a couple more things but I'll post those as issues.

File: intro_to_sf.Rmd (hunk 1 above)
Comment (jeanetteclark) [reply intro_to_sf.Rmd]: fixed!

**Pull request merged**